src/tests/test_control_flow/Makefile

```diff
@@ -0,0 +1,41 @@
+CC := clang
+LLC := ../../llvm/build/bin/llc
+CPOINTPASS := -Xclang -load -Xclang ../../passes/build/function_passes/libCheckPointPass.so
+UNOPTPASS := -Xclang -load -Xclang ../../passes/build/function_passes/libUnoptimizedCopyPass.so
+LIVEVARPASS := -Xclang -load -Xclang ../../passes/build/function_passes/libLiveVariablesPass.so
+MARKPASS := -Xclang -load -Xclang ./passes/build/basic_block_passes/libMarkUnoptimizedPass.so
+PASSFLAGS := $(CPOINTPASS) $(LIVEVARPASS) $(UNOPTPASS)
+STMAP_CHECKER_DIR := ../../stackmap_checker/
+OBJ_NAMES := utils.o stmap.o jump.o guard.o call_stack_state.o
+OBJS := $(foreach obj, $(OBJ_NAMES), $(STMAP_CHECKER_DIR)$(obj))
+PREFIX := trace
```

Comment:
Although it doesn't matter in this scenario, `PREFIX` is a special variable used in `install` targets. You probably want to avoid that name.

Reply:
Ah, I never knew that!